vxfreeswitch/voice.py

```diff
@@ -132,6 +132,10 @@ def unboundEvent(self, evdata, evname):
         log.msg("Unbound event %r" % (evname,))
 
 
+class ClientConnectError(Exception):
+    pass
```

Comment:
I usually replace ``pass`` with a docstring for exceptions (since it's the same number of lines and gives some indication of what sorts of situations the error is expected to be raised.